File: BOJ/[10250] ACM 호텔.js
let arr = require("fs")
  .readFileSync(process.platform === "linux" ? "/dev/stdin" : "input.txt")
  .toString()
  .trim()
  .split("\n")
  .map((i) => i.split(" ").map(Number));

// console.log("arr", ...arr);

for (let i = 1; i <= arr[0]; i++) {
  const H = arr[i][0];
  const W = arr[i][1];
  const N = arr[i][2];

  room = [N % H, Math.floor(N / H) + 1];

  if (room[0] == 0) {
    room[0] = H;
    room[1]--;
  }

  console.log(`${room[0]}${String(room[1]).padStart(2, "0")}`);
}

File: BOJ/[10250] ACM 호텔.py
T = int(input())
for _ in range(T):
    H, W, N = map(int, input().split())
    room = [N % H, N // H + 1]

    if room[0] == 0:
        room[0] = H
        room[1] -= 1

    print(f"{room[0]}{room[1]:02d}")